src/pages/AddData.tsx

```diff
@@ -67,10 +69,17 @@ const AddDataLoader = () => {
 
   useEffect(() => {
     document.title = 'Add Data - Philanthropy Data Commons';
+
     if (bulkUploadResponse?.entries) {
       setBulkUploads(bulkUploadResponse.entries);
     }
-  }, [bulkUploadResponse]);
+
+    const intervalId = setInterval(() => {
+      refreshBulkUploads();
+    }, REFRESH_BULK_UPLOADS_INTERVAL);
+
+    return () => clearInterval(intervalId);
```

Comment:
Super subtle point here but right now I think it is circumstantial that the return value of `clearInterval` happens to match the expected return value of the returned function of `useEffect`

Which is to say, this would be safer / more explicitly returning `void` if we wrote it as:

```
    return () => {
        clearInterval(intervalId);
    }
```
Comment

Reply:
(But truly great catch.)

Reply:
This makes sense! I was not thinking about the return value matching useEffect, seems like the right decision to add the block. I just added it!